backend/requirements.txt

```diff
@@ -1,17 +1,2 @@
-click==7.1.1
-Flask==1.1.1
-Flask-SQLAlchemy==2.4.1
-itsdangerous==1.1.0
-Jinja2==2.11.1
-MarkupSafe==1.1.1
-mne==0.19.2
-numpy==1.18.2
-pandas==1.0.3
-pandas-schema==0.3.5
-python-dateutil==2.8.1
-pytz==2019.3
-scipy==1.4.1
```

Comment:
We should only delete packages that are related to flask and sql

Reply:
idk if we are about to use all these other libraries